src/Api/Controllers/CiphersController.cs

```diff
@@ -360,6 +360,26 @@ public async Task DeleteMany([FromBody]CipherBulkDeleteRequestModel model)
             await _cipherService.DeleteManyAsync(model.Ids.Select(i => new Guid(i)), userId);
         }
 
+        [HttpDelete("admin")]
+        [HttpPost("delete-admin")]
+        public async Task DeleteManyAdmin([FromBody]CipherBulkDeleteRequestModel model)
+        {
+            if (!_globalSettings.SelfHosted && model.Ids.Count() > 500)
+            {
+                throw new BadRequestException("You can only delete up to 500 items at a time. " +
+                    "Consider using the \"Purge Vault\" option instead.");
```

Comment:
Is there actually a purge vault option for an organization's vault?

Reply:
Yep!

![Screen Shot 2020-07-20 at 12 09 19 PM](https://user-images.githubusercontent.com/15897251/87965750-dfa5d300-ca81-11ea-8b86-d25e5227db72.png)